src/main/java/com/github/maxopoly/environmentaleffects/datarepresentations/MobConfig.java

```diff
@@ -142,8 +145,10 @@ public MobConfig(String identifier, EntityType type, String name,
 
 	public Location findSpawningLocation(Location loc) {
```

Comment:
Yeah, it's ok. Thank you for the explanation, however! I'm also checking into Genesis' spawn code :)